src/main/java/com/org/checkr/service/JwtService.java

```diff
@@ -0,0 +1,73 @@
+package com.org.checkr.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtService {
+
+    public static  final String  SECRET = "e80XCtxHN9Siqh8S2IGlkeCmN0u8mImstqTNWBq64vJv2m1XY6RKCFQW6K0muoNQ";
```

Comment:
why SECRET key is in code? It should be stored in enviorment veriables.

Reply:
added it as an environment variable, thanks